thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/AnomalyResource.java

```diff
@@ -138,17 +120,29 @@ public AnomalyResource(AnomalyFunctionFactory anomalyFunctionFactory, AlertFilte
   /************** CRUD for anomalies of a collection ********************************************************/
   @GET
   @Path("/anomalies/view/{anomaly_merged_result_id}")
+  @ApiOperation(value = "Get anomalies")
   public MergedAnomalyResultDTO getMergedAnomalyDetail(
+      @ApiParam(value = "anomaly_merged_result_id",required = true)
       @NotNull @PathParam("anomaly_merged_result_id") long mergedAnomalyId) {
     return anomalyMergedResultDAO.findById(mergedAnomalyId);
   }
 
   // View merged anomalies for collection
   @GET
   @Path("/anomalies/view")
-  public List<MergedAnomalyResultDTO> viewMergedAnomaliesInRange(@NotNull @QueryParam("dataset") String dataset,
-      @QueryParam("startTimeIso") String startTimeIso, @QueryParam("endTimeIso") String endTimeIso,
-      @QueryParam("metric") String metric, @QueryParam("dimensions") String exploredDimensions,
+  @ApiOperation(value = "View merged anomalies for collection")
+  public List<MergedAnomalyResultDTO> viewMergedAnomaliesInRange(
+      @ApiParam(value = "dataset", required = true)
+      @NotNull @QueryParam("dataset") String dataset,
+      @ApiParam(value = "startTimeIso")
+      @QueryParam("startTimeIso") String startTimeIso,
+      @ApiParam(value = "endTimeIso")
```

Comment:
Is it end time exclusive or inclusive? Please refine this description and other descriptions of ApiParam.